### PR TITLE
Add steps to release docs for more clarity

### PR DESCRIPTION
**Pull request by Frost45**
Documenting scenario where changes are made to input and output test files like linearsvr. This in turn affects rsmcompare and rsmsummarize so the test files have to be updated.

### CODE REVIEW

File: doc/internal/release_process.rst [hunk 1]
```diff
@@ -3,10 +3,20 @@ RSMTool Release Process
...
 
+    .. note:: 
+
+        Some input files might be updated or changed due to changes in SKLL or scikit-learn packages, for example, the inputs and outputs for linearsvr. In this case, the tests need to be rerun and fixed. The process is similar to Example 2 under `Writing new functional tests <https://rsmtool.readthedocs.io/en/stable/contributing.html#writing-new-functional-tests>`_. You can also see `this pull request <https://github.com/EducationalTestingService/rsmtool/pull/525>`_ for more information. 
+
+    .. note:: 
+
+        Fairness test files have been excluded from the repository due to their non-deterministic nature so please do not add them back to the repository. More information on these files can be found in `this pull request <https://github.com/EducationalTestingService/rsmtool/pull/410>`_.
```
Comment (aloukina) [inline doc/internal/release_process.rst:18]:
```suggestion
        Several files have been excluded from the repository due to their non-deterministic nature so please do not add them back to the repository. The following files are currently excluded:
        * Fairness test files for `lr-eval-system-score-constant` test
        * Predictions and all evaluation files for `linearsvr` test. 
     
    Note that the full set of outputs from this test files are also used as input for `rsmcompare` and `rsmsummarize` tests. These *input* files need to be updated following the process under `Writing new functional tests <https://rsmtool.readthedocs.io/en/stable/contributing.html#writing-new-functional-tests>`_. You can also see `this pull request <https://github.com/EducationalTestingService/rsmtool/pull/525>`_ for more information. 
```

Comment (aloukina) [reply doc/internal/release_process.rst:18]: I think we want to combine these notes into one and re-structure a bit.

**Review by aloukina — CHANGES_REQUESTED** [review]
We need to edit the note a bit.

**Review by desilinguist — APPROVED** [review]
Really nice!!  👍

**Review by aloukina — APPROVED** [review]
Looks great, thank you for doing this!

**Pull request merged**